### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by Freakwill** [itdxer/neupy#237]
Got error when installing neupy. How do I fix it?

(with command `pip3 install neupy`)

> Collecting neupy
>   Using cached https://files.pythonhosted.org/packages/9c/a2/4d8dc5d686adcdd3ed2c98c85ccf69370ef0675574b0f8dbb9b91abc978c/neupy-0.8.0-py2.py3-none-any.whl
> Requirement already satisfied: numpy>=1.13.3 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from neupy) (1.16.2)
> Collecting tensorflow<=1.12.0,>=1.10.1 (from neupy)
>   Could not find a version that satisfies the requirement tensorflow<=1.12.0,>=1.10.1 (from neupy) (from versions: 1.13.0rc1, 1.13.0rc2, 1.13.1, 2.0.0a0)
> No matching distribution found for tensorflow<=1.12.0,>=1.10.1 (from neupy)

**Comment by itdxer** [comment]
From the log it looks like tensorflow >= 1.13 version is available, what command did you run?

**Comment by itdxer** [comment]
That's very odd, I'm not sure why do you get this error, since all TF versions are available in the pip

```
$ pip install tensorflow==0
Collecting tensorflow==0
  Could not find a version that satisfies the requirement tensorflow==0 (from versions: 0.12.1, 1.0.0, 1.1.0rc0, 1.1.0rc1, 1.1.0rc2, 1.1.0, 1.2.0rc0, 1.2.0rc1, 1.2.0rc2, 1.2.0, 1.2.1, 1.3.0rc0, 1.3.0rc1, 1.3.0rc2, 1.3.0, 1.4.0rc0, 1.4.0rc1, 1.4.0, 1.4.1, 1.5.0rc0, 1.5.0rc1, 1.5.0, 1.5.1, 1.6.0rc0, 1.6.0rc1, 1.6.0, 1.7.0rc0, 1.7.0rc1, 1.7.0, 1.7.1, 1.8.0rc0, 1.8.0rc1, 1.8.0, 1.9.0rc0, 1.9.0rc1, 1.9.0rc2, 1.9.0, 1.10.0rc0, 1.10.0rc1, 1.10.0, 1.10.1, 1.11.0rc0, 1.11.0, 1.12.0rc0, 1.12.0rc1, 1.12.0rc2, 1.12.0, 1.13.0rc0, 1.13.0rc1, 1.13.0rc2, 1.13.1, 2.0.0a0)
```

FYI, I created new release (v0.8.1) which will include support for the tensorflow 1.13. You can try to install it from the library: https://github.com/itdxer/neupy/pull/238

**Comment by itdxer** [comment]
Also, can you try to install tensorflow manually?

```
pip install tensorflow==1.12.0
```

**Comment by itdxer** [comment]
FYI, I just published v0.8.1 release that includes support for TF 1.13 . You might try to install it as well
